data-serving/data-service/src/controllers/case.ts

```diff
@@ -129,108 +134,54 @@ export class CasesController {
             }
 
             let columnsList: string[] = [];
-
-            switch (req.body.format) {
-                case 'csv':
-                    res.setHeader('Content-Type', 'text/csv');
-                    await axios.get<string>(CaseFieldTextFile)
-                    .then((txtRes) => {
-                        columnsList = txtRes.data.split('\n');
-                        const columnsString = columnsList.join();
-                        res.write(columnsString);
-                        res.write('\n');
-                    });
-
-                    try {
-                        let doc: CaseDocument;
-                        doc = await cursor.next();
-                        while (doc != null) {
-                            delete doc.restrictedNotes;
-                            const parsedCase = parseDownloadedCase(doc);
-                            const stringifiedCase = stringify([parsedCase], {
-                                header: false,
-                                columns: columnsList,
-                            });
-                            res.write((stringifiedCase));
-                            doc = await cursor.next();
-                        }
-                    } catch(err) {
-                        logger.error(`Error streaming case data: ${err}`);
-                        res.status(500).json('A server error occurred while streaming case data');
-                        return;
-                    }
-                    res.end();
-                
-                    break;
-
-                case 'tsv':
-                    res.setHeader('Content-Type', 'text/tsv');
-                    await axios.get<string>(CaseFieldTextFile)
-                    .then((txtRes) => {
-                        columnsList = txtRes.data.split('\n');
-                        const columnsString = columnsList.join('\t');
-                        res.write(columnsString);
-                        res.write('\n');
+            let doc: CaseDocument;
+
+            if (req.body.format == 'csv' || req.body.format == 'tsv') {
+                res.setHeader('Content-Type', `text/${req.body.format}`);
+                let delimiter: string = (req.body.format == 'tsv') '\t' : ',';
```

Comment:
ternary op missing ?

Reply:
yep, not sure how that happened, but caught it during testing